contracts/cw20-merkle-airdrop/src/contract.rs

```diff
@@ -124,12 +140,25 @@ pub fn execute_register_merkle_root(
 
 pub fn execute_claim(
     deps: DepsMut,
-    _env: Env,
+    env: Env,
     info: MessageInfo,
     stage: u8,
     amount: Uint128,
     proof: Vec<String>,
 ) -> Result<Response, ContractError> {
+    // airdrop begun
```

Comment:
I would pull out these lines (the green diff) into a function like:

`is_active(deps: Deps, env: Env, stage: u8) -> StdResult<bool> {}`

which you can call here and easily unit test. You may well want to use the same check somewhere else later.